velox/common/memory/Memory.h

```diff
@@ -79,40 +87,23 @@ struct MemoryManagerOptions {
   /// Specifies the backing memory allocator.
   MemoryAllocator* allocator{MemoryAllocator::getInstance()};
 
-  /// ================== 'MemoryArbitrator' settings ==================
-
-  MemoryArbitrator::Kind arbitratorKind{MemoryArbitrator::Kind::kNoOp};
```

Comment:
Let's keep the other options?

Reply:
done, added `MemoryArbitratorFactory` to the config.